### PR TITLE
refacto(environments): merge ui and feature libs into one

### PR DESCRIPTION
**Pull request by WilliamTraoreee**
# What does this PR do?

> Link to the JIRA ticket

Put description here

> Screenshot of the feature

Put screenshot following the template `before` => `after` here

---

### Global

- [ ] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [ ] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [ ] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - refacto(environments): merge ui and feature libs into one - 2022-06-16T13:35:13Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/26865077-3d42-4d3b-bf5e-e01c4fd46bf6/applications)

File: libs/pages/environments/src/lib/feature/page-edit-deployment-rule-feature/page-edit-deployment-rule-feature.tsx [hunk 1]
```diff
@@ -10,8 +9,9 @@ import { useEffect } from 'react'
 import { BaseLink } from '@console/shared/ui'
 import { ENVIRONMENTS_DEPLOYMENT_RULES_URL, ENVIRONMENTS_URL } from '@console/shared/router'
 import { dateToHours } from '@console/shared/utils'
+import PageEditDeploymentRule from '../../ui/page-edit-deployment-rule/page-edit-deployment-rule'
 
-export function EditDeploymentRule() {
+export function PageEditDeploymentRuleFeature() {
```
Comment (bdebon) [inline libs/pages/environments/src/lib/feature/page-edit-deployment-rule-feature/page-edit-deployment-rule-feature.tsx:14]: Are these really pages or just component? I don' remember a dedicated route for this. Tell me if I'm wrong

**Review by bdebon — COMMENTED** [review]
Just a very short question for something I'm not sure about at all. Otherwise everything is good!

File: libs/pages/environments/src/lib/feature/page-edit-deployment-rule-feature/page-edit-deployment-rule-feature.tsx (hunk 1 above)
Comment (WilliamTraoreee) [reply libs/pages/environments/src/lib/feature/page-edit-deployment-rule-feature/page-edit-deployment-rule-feature.tsx:14]: Yes they are pages, we just removed the routes for now

**Pull request merged**